### DISCUSSION TITLE
CESM1.2-CARMA with nitrate and ammonium

### DISCUSSION THREAD

**Discussion by tilmes** [NCAR/CARMA_dev#20] [trop_strat CARMA model]
@pengfei-aerosol @cbardeen @Toon-cloud 
I started a new discussion on this topic, so we can focus here on the nitrate and ammonium model

@pengfei-aerosol post:
I did some diagnose on my version, CESM1.2-CARMA with nitrate and ammonium,

here is only one month (June), and the CARMA budget is nearly balanced to me. I include budget for H2SO4, HNO3 and NH3 gases.
<img width="865" alt="image" src="https://user-images.githubusercontent.com/22222754/216632274-61b5bc3a-165d-44c5-8aca-e5c2b4ae4cdc.png">

**Comment by pengfei-aerosol** [comment]
updated yearly averaged budget for CARMA processes:

![cesm1 2-carma_budget](https://user-images.githubusercontent.com/124237561/217489538-37aa0b50-04cc-41af-af14-057a180ad36b.png)

**Comment by pengfei-aerosol** [comment]
NO3/NH4 burden compared with CESM-MOSAIC (Zaveri et al., 2021)
![Picture2](https://user-images.githubusercontent.com/124237561/217492257-c920e69e-8f73-4a44-b8a1-8d004de2c9e6.png)
